.github/workflows/release.yml

```diff
@@ -18,31 +18,42 @@ env:
   AWS_S3_BUCKET_NAME: "nr-downloads-main"
   AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock"
   AWS_REGION: "us-east-1"
+  AWS_ACCESS_KEY_ID: ${{ secrets.OHAI_AWS_ACCESS_KEY_ID_PRODUCTION }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.OHAI_AWS_SECRET_ACCESS_KEY_PRODUCTION }}
+  AWS_ROLE_ARN: ${{ secrets.OHAI_AWS_ROLE_ARN_PRODUCTION }}
+  AWS_ROLE_SESSION_NAME: ${{ secrets.OHAI_AWS_ROLE_SESSION_NAME_PRODUCTION }}
 
 jobs:
   publishing-to-s3-linux:
     name: Publish linux artifacts into s3 prod bucket
     runs-on: ubuntu-20.04
 
+    strategy:
+      matrix:
+        assetsType:
+          - "targz"
+          - "deb"
+          - "rpm"
```

Comment:
aren't we missing docker, zip & msi here?

Reply:
they are in another job section